server/swap/swap.go

```diff
@@ -1502,7 +1502,7 @@ func (s *Swapper) processInit(msg *msgjson.Message, params *msgjson.Init, stepIn
 			stepInfo.match.ID(), actor, params.CoinID, params.Contract, err)
 		actor.status.mtx.RUnlock()
 		s.respondError(msg.ID, actor.user, msgjson.ContractError,
-			"redemption error")
+			fmt.Sprintf("contract error encountered: %v", err))
```

Comment:
Good catch!  That's not a redemption error lol.  In this case I don't see the harm in including the full `err`.